<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>标准</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>项目符号(●)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>项目符号(○)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>项目符号(1.2.3.)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>项目符号(a.b.c.)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>项目符号(A.B.C.)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>项目符号(ⅰ.ⅱ.ⅲ)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="sizeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="printAction"/>
    <addaction name="printPreviewAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu_D">
     <property name="title">
      <string>字体(D)</string>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underLineAction"/>
    </widget>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="leftAlineAction"/>
     <addaction name="centerAction"/>
     <addaction name="rightAlineAction"/>
     <addaction name="justifyAction"/>
    </widget>
    <addaction name="menu_D"/>
    <addaction name="menu"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗体(W)</string>
    </property>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="printAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="colorAction"/>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underLineAction"/>
   <addaction name="separator"/>
   <addaction name="leftAlineAction"/>
   <addaction name="centerAction"/>
   <addaction name="rightAlineAction"/>
   <addaction name="justifyAction"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="statusTip">
    <string>新建一个文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="statusTip">
    <string>打开一个已经存在的文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="statusTip">
    <string>保存到当前文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为(A)</string>
   </property>
   <property name="statusTip">
    <string>将当前内容保存为一个新的文件</string>
   </property>
  </action>
  <action name="printAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(P)</string>
   </property>
   <property name="statusTip">
    <string>打印当前文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="printPreviewAction">
   <property name="text">
    <string>打印预览</string>
   </property>
   <property name="statusTip">
    <string>预览打印效果</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="statusTip">
    <string>撤销当前操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(R)</string>
   </property>
   <property name="statusTip">
    <string>重复上一次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="statusTip">
    <string>将内容剪切到剪切板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="statusTip">
    <string>复制当前内容到剪切板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="statusTip">
    <string>将剪切板内容粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色(C)</string>
   </property>
   <property name="statusTip">
    <string>改变字体颜色</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(B)</string>
   </property>
   <property name="statusTip">
    <string>给文字加粗</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(I)</string>
   </property>
   <property name="statusTip">
    <string>使文本倾斜</string>
   </property>
  </action>
  <action name="underLineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/underline.png</normaloff>:/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线(U)</string>
   </property>
   <property name="statusTip">
    <string>给文本添加下划线</string>
   </property>
  </action>
  <action name="leftAlineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/align-left.png</normaloff>:/images/align-left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐(L)</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Center.png</normaloff>:/images/Center.png</iconset>
   </property>
   <property name="text">
    <string>居中(C)</string>
   </property>
  </action>
  <action name="rightAlineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/align-right.png</normaloff>:/images/align-right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐(R)</string>
   </property>
  </action>
  <action name="justifyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/justify.png</normaloff>:/images/justify.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭(O)</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有(A)</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个(X)</string>
   </property>
   <property name="statusTip">
    <string>将焦点切换到下一个子窗体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>前一个(V)</string>
   </property>
   <property name="statusTip">
    <string>将焦点切换到前一个子窗体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(A)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
